.circleci/config.yml

```diff
@@ -216,7 +216,8 @@ executors:
       - image: cimg/rust:1.50.0
   macos:
     macos:
-      xcode: "12.3.0"
+      xcode: 12.5.0
```

Comment:
@st3fan if we do switch to this configuration, will firefox-ios be able to consume a library built with this version as well as focus?